extra/mondoo-asset-count.mql.yaml

```diff
@@ -92,242 +92,242 @@ packs:
-        mql: aws.ecr.publicRepositories.length
-
-      - uid: mondoo-asset-count-aws-ecr-container-images
-        title: AWS Elastic Container Images (ECR)
-        mql: aws.ecr.images.length
-
-      - uid: mondoo-asset-count-aws-rds-dbclusters
-        title: AWS RDS Database Clusters
-        mql: aws.rds.dbClusters.length
-
-      - uid: mondoo-asset-count-aws-cloudtrails
-        title: AWS CloudTrails
-        mql: aws.cloudtrail.trails.length
-
-      - uid: mondoo-asset-count-aws-dynamodb-tables
-        title: AWS DynamoDB Tables
-        mql: aws.dynamodb.tables.length
-
-      - uid: mondoo-asset-count-aws-dynamodb-global-tables
-        title: AWS DynamoDB Global Tables
-        mql: aws.dynamodb.globalTables.length
-
-      - uid: mondoo-asset-count-aws-ecs-clusters
-        title: AWS ECS Clusters
-        mql: aws.ecs.clusters.length
-
-      - uid: mondoo-asset-count-aws-ecs-container-instances
-        title: AWS ECS Container Instances
-        mql: aws.ecs.containerInstances.length
-
-      - uid: mondoo-asset-count-aws-ecs-containers
-        title: AWS ECS Containers
-        mql: aws.ecs.containers.length
-
-      - uid: mondoo-asset-count-aws-efs-filesystems
-        title: AWS EFS Filesystems
-        mql: aws.efs.filesystems.length
-
-      - uid: mondoo-asset-count-aws-elasticache-clusters
-        title: AWS ElastiCache Clusters
-        mql: aws.elasticache.clusters.length
-
-      - uid: mondoo-asset-count-aws-elasticache-cache-clusters
-        title: AWS ElastiCache Cache Clusters
-        mql: aws.elasticache.cacheClusters.length
-
-      - uid: mondoo-asset-count-aws-elb-application
-        title: AWS Elastic Application Load Balancers
-        mql: aws.elb.loadBalancers.length
-
-      - uid: mondoo-asset-count-aws-elb-classic
-        title: AWS Elastic Classic Load Balancers
-        mql: aws.elb.classicLoadBalancers.length
-
-      - uid: mondoo-asset-count-aws-emr-clusters
-        title: AWS Elastic Map Reduce Clusters
-        mql: aws.emr.clusters.length
-
-      - uid: mondoo-asset-count-aws-es-domains
-        title: AWS Elasticsearch Service Domain
-        mql: aws.es.domains.length
-
-      - uid: mondoo-asset-count-aws-guardduty-detectors
-        title: AWS Guard Duty Detectors
-        mql: aws.guardduty.detectors.length
-
-      - uid: mondoo-asset-count-aws-kms-keys
-        title: AWS KMS Keys
-        mql: aws.kms.keys.length
+queries:
+  - uid: mondoo-asset-count-on-vsphere-cluster-esxi
+    title: Retrieve all ESXi hosts
+    query: |
+      vsphere.datacenters { hosts.length }
+
+  - uid: mondoo-asset-count-on-vsphere-cluster-vms
+    title: Retrieve all VMs from vSphere cluster
+    query: |
+     vsphere.datacenters { vms.length }
+
+  - uid: mondoo-asset-count-on-azure
+    title: Azure virtual machine count
+    query: |
+      azure.subscription.compute.vms.length
+
+  - uid: mondoo-count-users-in-azure
+    title: Azure user count
+    query: |
```

Comment:
Just wondering:
In this PR, we have `mql` and `query`. But in the associated PR, you replaced `query` with `mql` in the testdata.

Reply:
I didn't see this is using `query`, i will change it, great catch!